logic/labrynna.go

```diff
@@ -20,11 +20,13 @@ var labrynnaNodes = map[string]*Node{
 	"lynna city chest": OrSlot("ember seeds", "currents"),
 	"shore present": Or("flute", "ricky's gloves",
 		And("break bush", "feather"), And("lynna city", "bracelet"),
-		And("currents", Or("feather", "flippers", "raft"))),
+		And("currents", Or("feather", "flippers", "raft")),
+		And("ages", "break bush"),
+		And("lynna city", "mermaid suit")),
 	"south shore dirt": AndSlot("shore present", Or("shovel", "flute")),
 	"balloon guy": And("feather", Or("sword", "boomerang"),
-		Or("currents", And(Or("break bush", "shore present"),
-			Or("any seed shooter", "ricky's gloves", "ricky's flute")))),
+		Or("currents", "ricky's gloves", "ricky's flute",
+			And(Or("break bush", "flute", "shore present"), "any seed shooter"))),
```

Comment:
This should just be `And("shore present", "any seed shooter")`; "shore present" covers the break bush and flute cases.